Formula/aws-nuke.rb

```diff
@@ -0,0 +1,26 @@
+require "etc"
```

Comment:
Why `require "etc"`?

Reply:
not required. left over from previous version.